src/Modal.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ModalContainer = styled.div`
+  position: fixed;
+  top: 0px;
+  right: 0px;
+  bottom: 0px;
+  left: 0px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 10;
+`;
+
+const ModalBackground = styled.div`
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  bottom: 0px;
+  left: 0px;
+  background-color: #000;
+  opacity: 0.8;
+`;
+
+const ModalContent = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  opacity: 1;
+  padding: 2em;
+  padding-bottom: 3em;
+  border-radius: 0.25rem;
+`;
+
+const ModalHeader = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+`;
+
+const ModalCloseButton = styled.button`
+  margin: 0px;
+  padding: 0px;
+  cursor: pointer;
+  background: none;
+  border: none;
+`;
```

Comment:
Just re-iterating it might be cool to have the styles in the bottom 😇

Reply:
Cooled 👍